pkg/curatedpackages/packagecontrollerclient_test.go

```diff
@@ -149,7 +161,7 @@ func newPackageControllerTests(t *testing.T) []*packageControllerTest {
 			kubectl:        k,
 			chartInstaller: ci,
 			command: curatedpackages.NewPackageControllerClient(
-				ci, k, clusterName, kubeConfig, chartDev,
+				ci, k, clusterName, nil, kubeConfig, chartDev,
```

Comment:
From usage, looks like clusterSpec is not used by all. Should we make it an option in PCC?

Reply:
it is only used in the generate yaml method. I don't want to create one for tests that don't use it.

Reply:
What I am referring to is in the NewPackageControllerClient should we make the clusterSpec a PackageControllerClientOpt so that you don't need to pass the nil everywhere.

Reply:
oooh